src/enhanced-textarea.jsx

```diff
@@ -2,6 +2,28 @@ const React = require('react');
 const StylePropable = require('./mixins/style-propable');
 const AutoPrefix = require('./styles/auto-prefix');
 
+const rowsHeight = 24;
+
+const styles = {
```

Comment:
@shaurya947 @oliviertassinari I think declaring the constant in this way is fine here. Since the styles object doesn't depend on the instance of the component, declaring the constant outside is a little more efficient.